buildSrc/src/main/groovy/local.java.vro-dep-constraints.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
         implementation "org.springframework:spring-webmvc:${spring_framework_version}"
         implementation "org.springframework:spring-beans:${spring_framework_version}"
         implementation "io.github.classgraph:classgraph:${class_graph_version}"
+        implementation "com.fasterxml.jackson.core:jackson-databind:${jackson_databind_version}"
```

Comment:
Add dependency constraints to this file rather than many files.

@sethdarragile6 Please include this in your wiki documentation.